src/main/java/org/mockito/internal/stubbing/defaultanswers/ReturnsEmptyValues.java

```diff
@@ -109,8 +112,20 @@ Object returnValueFor(Class<?> type) {
             return new LinkedHashMap<Object, Object>();
         } else if ("java.util.Optional".equals(type.getName())) {
             return JavaEightUtil.emptyOptional();
+        } else if ("java.util.OptionalDouble".equals(type.getName())) {
```

Comment:
To be honest, I had not realised this. Especially because I dont think someone relies on the previous null behavior.

Nonetheless we should have discussed this, good point.